projects/lights_control/inc/lights_signal_fsm.h

```diff
@@ -11,14 +11,22 @@
 #include "lights_events.h"
 
 // Instance of lights signal.
+
 typedef struct LightsSignalFsm {
   LightsBlinker blinker;
   FSM fsm;
 } LightsSignalFsm;
 
-// Initializes state machines, and blinker.
+// Initializes the FSM with a non-syncing blinker.
+#define lights_signal_fsm_init_front(fsm, duration) lights_signal_fsm_init(fsm, duration, 0)
```

Comment:
You shouldn't need to distinguish between front/back in this module. Please use a #define instead of 0.

Reply:
🅱️ool.